src/launch/cli/service/build/__init__.py

```diff
@@ -53,6 +51,10 @@
     default=False,
     help="(Optional) Skip cloning the application files. Will assume you're in a directory with the application files.",
 )
+@click.option(
+    "--registry-type",
+    help="(Required) Based off of the registry-type value, the sequence of make commands used before build is decided.",
+)
```

Comment:
Can you provide the allowed types - docker, npm etc. that will be helpful for users

Reply:
fixed!!